internal/kafka/internal/services/cluster_placement_strategy.go

```diff
@@ -150,29 +149,31 @@ func (f *FirstReadyWithCapacity) FindCluster(kafka *dbapi.KafkaRequest) (*api.Cl
 	}
 
 	clusters, findAllClusterErr := f.ClusterService.FindAllClusters(criteria)
-	if findAllClusterErr != nil || len(clusters) == 0 {
+	if findAllClusterErr != nil {
```

Comment:
Why did you remove this again?

Reply:
I had to remove this due to a previous commit when I changed the return type of `FindAllClusters` but the cluster placement strategy still returns a `ServiceError` so i needed to cast it, which caused tests to fails. 

I'm supposed to re-add this, good spot!